windows/src/engine/kmtip/kmtip.cpp

```diff
@@ -169,7 +169,8 @@ STDAPI_(ULONG) CKMTipTextService::Release()
 
```

Comment:
Does `LogEnter();` go here?

Reply:
No, I don't want log events for every interface AddRef and Release -- really noisy. Only in the error case as shown.